portfolio/src/main/webapp/script.js

```diff
@@ -154,3 +170,36 @@ function deleteComment(commentId) {
         LoadComments(0);
     });
 }
+
+/**
+ * If the user is logged in, the function inits loggedInElements
+ * Else the function inits loggedOutElements
+ */
+function initUserLoggedElements() {
+    // Make a GET request to "/user" to get user information in user object
+    fetch('user').then(response => response.json()).then((user) => {
+        if(user.loggedIn === true) {
+            const logoutLink = document.getElementById('logout-link');
+            logoutLink.href = user.logoutURL;
+            displayElements(loggedInElements);
+        }
+        else {
```

Comment:
Move this else up to maintain consistency. (you've used this in other files)
if(){
} else {
}

Reply:
Thanks!